classifier_train.py

```diff
@@ -0,0 +1,87 @@
+import torch
+import torch.nn as nn
+from pdd.model import PDDModel
+from pdd.data_utils import unzip_data
+from pdd.data_utils import load_config
+from pdd.model import get_trained_model
+from train import prepare_datasets
+from train import fix_random_seed
+from train import split_on_train_and_test
```

Comment:
if you are using these functions more than once, then move they to `pdd.utils` for convenience